ush/python/pygfs/task/atmens_analysis.py

```diff
@@ -290,15 +288,15 @@ def jedi2fv3inc(self: Analysis) -> None:
 
             # make output directory for member increment
             incdir = [
-                os.path.join(self.task_config.COMOUT, memchar, 'atmos')
+                os.path.join(self.task_config.COM_TOP_ENS, memchar, 'analysis/atmos')
```

Comment:
Modify these similarly to above, using the appropriate template. Basically, usage of `COM_TOP` should just be avoided. It's only being used in two places where we need to cut above the member level.

Reply:
Requested change committed at [`dc0f196`](https://github.com/RussTreadon-NOAA/global-workflow/commit/dc0f1963c9088ed946661c29e196c6fba42ecf5c)